automobile/i18n/automobile.pot

```diff
@@ -0,0 +1,184 @@
+#: model_terms:ir.ui.view,arch_db:automobile.view_x_brands_form
+msgid "Brands"
+msgstr ""
+
+#. module: automobile
+#: model:ir.model.fields,field_description:automobile.field_x_brands_x_color
+msgid "Color"
+msgstr ""
+
+#. module: automobile
+#: model_terms:ir.actions.act_window,help:automobile.action_brands
+msgid "Create Brands."
+msgstr ""
+
+#. module: automobile
+#: model:ir.model.fields,field_description:automobile.field_x_brands_x_name
+msgid "Description"
+msgstr ""
+
+#. module: automobile
+#: model:product.template,name:automobile.product_template_10
+msgid "Drum Brake"
+msgstr ""
+
+#. module: automobile
+#: model:product.attribute.value,name:automobile.product_attribute_value_6
+msgid "Engine"
+msgstr ""
+
+#. module: automobile
+#: model:ir.actions.server,name:automobile.update_the_warranty__f6563e1c-89cd-4350-b2c5-0558e726fded
+msgid "Execute Code"
+msgstr ""
+
+#. module: automobile
+#: model:ir.model.fields,field_description:automobile.field_stock_move_line_x_expiry_date
+#: model:ir.model.fields,field_description:automobile.field_stock_move_x_expiry_date
+#: model_terms:ir.ui.view,arch_db:automobile.view_stock_move_line_detailed_operation_tree_inherit_automobile
+#: model_terms:ir.ui.view,arch_db:automobile.view_stock_move_tree_inherit_automobile
+msgid "Expiry Date"
+msgstr ""
+
+#. module: automobile
+#: model:product.template,name:automobile.product_template_12
+msgid "Filter"
+msgstr ""
+
+#. module: automobile
+#: model:product.attribute.value,name:automobile.product_attribute_value_2
+msgid "Ford"
+msgstr ""
+
+#. module: automobile
+#: model:product.template,name:automobile.product_template_9
+msgid "Front Brake Pads"
+msgstr ""
+
+#. module: automobile
+#: model:product.template,name:automobile.product_template_8
+msgid "Gift Card"
+msgstr ""
+
+#. module: automobile
+#: model:product.attribute.value,name:automobile.product_attribute_value_3
+msgid "KIA"
+msgstr ""
+
+#. module: automobile
+#: model:product.attribute.value,name:automobile.product_attribute_value_4
+msgid "MG"
+msgstr ""
+
+#. module: automobile
+#: model_terms:ir.ui.view,arch_db:automobile.view_x_brands_form
+msgid "Name..."
+msgstr ""
+
+#. module: automobile
+#: model:ir.model.fields,field_description:automobile.field_product_template_x_oem_no
+msgid "OEM No."
+msgstr ""
+
+#. module: automobile
+#: model:product.attribute.value,name:automobile.product_attribute_value_7
+msgid "Oil Filter"
+msgstr ""
+
+#. module: automobile
+#: model_terms:ir.ui.view,arch_db:automobile.product_template_only_form_view_inherit_automobile
+msgid "Part Number"
+msgstr ""
+
+#. module: automobile
+#: model:product.template,name:automobile.product_template_11
+msgid "Rear Brake Pads"
+msgstr ""
+
+#. module: automobile
+#: model:ir.model.fields,field_description:automobile.field_x_brands_x_sequence
+msgid "Sequence"
+msgstr ""
+
+#. module: automobile
+#: model:product.attribute.value,name:automobile.product_attribute_value_5
+msgid "TATA"
```

Comment:
Wait, what? ^^'

Reply:
It is the name of a car brand the car brake are sold for. No really for translation